x-pack/dockerlogbeat/scripts/bundle.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+
+#///////////////////////////////////////////
+# Global Variables
+#///////////////////////////////////////////
+
+PLUGIN_NAME="elastic/elastic-logging-plugin"
+CONTAINER_ARTIFACT_NAME="ossifrage/rootfsimage"
+CONTAINER_NAME="plugin_container"
+ROOTFS_EXPORT_NAME="tmp_out.tar"
+ROOTFS_NAME="rootfs"
+VERSION=""
+PLUGIN_FULL=""
+TARGET_TO_COPY=""
+
+
+#///////////////////////////////////////////
+# Helper Functions
+#///////////////////////////////////////////
+
+
+cleanup(){
+    echo "Cleaning up..."
+    if [ -e "$ROOTFS_EXPORT_NAME" ]; then
+    rm $ROOTFS_EXPORT_NAME
+    fi
+    if [ -e "$ROOTFS_NAME" ]; then
+    rm -rf $ROOTFS_NAME
+    fi
+    if [ -e "config.json" ]; then
+    rm config.json
+    fi
+    docker container rm $CONTAINER_NAME
+    docker image rm $CONTAINER_ARTIFACT_NAME
+}
+
+# cli_help prints  --help
+cli_help(){
+    cli_name=${0##*/}
+
+    echo "
+$cli_name
+Utility to import file into the Elastic Log Driver
+
+Usage: $cli_name [FILE OR DIRECTORY]
+"
+exit 1
+}
+
+# run_docker_cmds performs the docker operations to pull and export the container
+run_docker_cmds(){
+
+    echo "Downloading container..."
+    docker pull -q $CONTAINER_ARTIFACT_NAME
+
+
+    echo "Creating container..."
+    docker container create --name plugin_container $CONTAINER_ARTIFACT_NAME
+
+    echo "Exporting...."
+    mkdir -p $ROOTFS_NAME
+    docker export -o $ROOTFS_EXPORT_NAME $CONTAINER_NAME
+}
+
+# build_plugin builds the plugin from the tar file produced by docker export
+build_plugin(){
+    echo "Building plugin...."
+
+    tar xf $ROOTFS_EXPORT_NAME -C rootfs
+
+    echo "Adding $TARGET_TO_COPY..."
+
+    cp -r "$TARGET_TO_COPY" "$ROOTFS_NAME"
+
+    echo "Creating new plugin..."
+    wget -q https://raw.githubusercontent.com/elastic/beats/master/x-pack/dockerlogbeat/config.json 
```

Comment:
Do we always want `config.json` from master? How about adding the version to the URL?